flaskr_tests.py

```diff
@@ -133,6 +121,71 @@ def test_move(self):
         assert '1' in id_order[0]
         assert '2' in id_order[1]
 
+    def test_unittest_move_entry(self):
+        """Test entries are moved in the database."""
+        with flaskr.app.app_context():
+            flaskr.init_db()
```

Comment:
It is not really clear what is going on here, it would be better to have the explicit setup.